mspray/apps/reveal/tests/test_utils.py

```diff
@@ -12,13 +12,15 @@
 REVEAL_SPRAY_STATUS_FIELD = "task_business_status"
 REVEAL_NOT_SPRAYABLE_VALUE = "Not Sprayable"
 REVEAL_SPRAYED_VALUE = "Sprayed"
+REVEAL_NOT_SPRAYED_VALUE = "Not Sprayed"
```

Comment:
Why not use the existing `MSPRAY_WAS_NOT_SPRAYED_VALUE`? We end up with twice the number of settings which communicate the same thing. Or are you anticipating that reveal mspray and irs mspray will be deployed on the same database and concurrently receiving data?

Reply:
Initially I wanted it to be completely independent, to the extent that it could be deployed on the same server as regular mSpray.  But this is a fair point.  I'll address it in a future PR